lib/rails_util/timezone_helper.rb

```diff
@@ -26,7 +26,8 @@ def convert_timezone(time, to_timezone, from_timezone, **options)
       # @param [String] to_timezone the `ActiveSupport::TimeZone` `timezone.tzinfo.name` of the desired timezone
       # @param [String] to_timezone the `ActiveSupport::TimeZone` `timezone.tzinfo.name` of the timezone being converted
       # @return ActiveSupport::Duration object as the number of seconds between the given timezones
-      def offset_in_seconds(time, to_timezone, from_timezone)
+      def offset_in_seconds(to_timezone, from_timezone)
+        time = Time.now
```

Comment:
Does this fix a bug or is it just a housekeeping change?

Reply:
When working locally, when calling `utc_offset` on my `time` object was giving me the wrong object (not taking into account daylight savings). That was the whole thing I was trying to fix in this PR. for whatever reason calling `.utc_offset` on `Time.now` is grabbing the correct offset with daylight savings. I found a stackoverflow about it and should have linked it.

Reply:
Ah gotcha. Is that something you could add a spec to cover? Or was it a weird ephemeral thing?

Reply:
it's an ephemeral thing but there is this spec that was trying to partially catch this: https://github.com/LaunchPadLab/rails_util/blob/883a05dd2766eaf2d6c2e56dcf65ed771733a78f/spec/rails_util/timezone_helper_spec.rb#L31

Reply:
I think this helps points to the difference: https://rails.lighthouseapp.com/projects/8994/tickets/3037-timecurrentutc_offset-and-timezoneutc_offset-are-different-during-dst

In Empower I added this updated code locally, and in my tests, I was providing timezones for my time objects, which made them TimeWithZone objects. calling `.utc_offset` on these provide different values during DST then simple time objects, and my specs were failing.

Using `Time.now` in this module to only capture the offset to me builds confidence that DST is properly accounted for.

Reply:
👍now that this change has been made, can the mock for `utc_offset` be removed?

Reply:
it's not mocking it - it's just grabing the offset for the timezone in question (to_timezone) and converting it to hours. `offset_in_seconds` is the delta between the timezones you're converting. I renamed the methods to make that more apparent.